src/Microsoft.ML.PCA/PCACatalog.cs

```diff
@@ -40,10 +40,10 @@ internal static PrincipalComponentAnalyzer ProjectToPrincipalComponents(this Tra
             => new PrincipalComponentAnalyzer(CatalogUtils.GetEnvironment(catalog), columns);
 
         /// <summary>
-        /// Trains an approximate principal component analysis (PCA) model using randomized SVD algorithm.
+        /// Create <see cref="RandomizedPcaTrainer"/>, which trains an approximate principal component analysis (PCA) model using randomized SVD algorithm.
```

Comment:
Should we spell out SVD as singular value decomposition (SVD) like we do with PCA?

Reply:
+1

Reply:
yep